core/src/main/java/com/google/zxing/aztec/encoder/State.java

```diff
@@ -165,5 +168,17 @@ BitArray toBitArray(byte[] text) {
   public String toString() {
     return String.format("%s bits=%d bytes=%d", HighLevelEncoder.MODE_NAMES[mode], bitCount, binaryShiftByteCount);
   }
+  
+  private static int calculateBinaryShiftCost(State state) {
+      if (state.binaryShiftByteCount > 62) {
```

Comment:
Nit: use 2-space indent here. Also I think you can dispense with the "else" statements and just write:

```
if (...) {
   return ...;
}
if (...) {
   return ...;
}
```